docs/content/classic/index.md

```
@@ -32,12 +32,11 @@ disciplines, as long as it can be done on a computer.
 
 Welcome to the AutoGPT Classic Documentation.
 
-The AutoGPT Classic project consists of four main components:
+The AutoGPT Classic project consists of three main components:
 
 - The [Agent](#agent) &ndash; also known as just "AutoGPT Classic"
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)